admin/shared/form/interceptor/form-errors.interceptor.ts

```diff
@@ -35,27 +38,41 @@ export class FormErrorsInterceptor implements NestInterceptor {
 
     return next.handle().pipe(
       catchError(error => {
-        const parsedError = JSON.parse(error.message) as Array<{
-          property: string;
-          constraints: string[];
-        }>;
-        // In case of validation error, we render the redirect with the flashed errors and DTO
-        return of(parsedError).pipe(
-          map(validationErrors => {
-            return validationErrors.reduce(
-              (acc, currentValidationError) => ({
-                ...acc,
-                [currentValidationError.property]:
-                  currentValidationError.constraints,
+        let errorMessage;
+        try {
+          errorMessage = JSON.parse(error.message);
```

Comment:
question: is the JSON.parse still needed ?

Reply:
cf. below